src/table/body/table-row.component.ts

```diff
@@ -28,18 +28,24 @@ import { TableRowSize } from "../table.types";
 				[headers]="model.getHeaderId('expand')"
 				(expandRow)="expandRow.emit()">
 			</td>
-			<td
-				*ngIf="!skeleton && showSelectionColumn && !enableSingleSelect"
-				ibmTableCheckbox
-				class="bx--table-column-checkbox"
-				[size]="size"
-				[selected]="selected"
-				[label]="getCheckboxLabel()"
-				[row]="row"
-				[skeleton]="skeleton"
-				[headers]="model.getHeaderId('select')"
-				(change)="onSelectionChange()">
-			</td>
+			<ng-container *ngIf="!skeleton && showSelectionColumn && !enableSingleSelect">
+				<td
+					*ngIf="!showSelectionColumnCheckbox; else tableCheckboxTemplate">
```

Comment:
Not sure if it's related to this, but if you turn off `showSelectionColumn`, there is still an extra `td` that shows up in expanded rows with more than one column.

![image](https://user-images.githubusercontent.com/45505172/106636062-8732b080-654f-11eb-816a-3e93c9564724.png)

Reply:
It _is_ related. Good catch!